core/src/validation.jl

```diff
@@ -86,6 +86,17 @@ n_neighbor_bounds_control(::Val{:FlowDemand}) = n_neighbor_bounds(0, 0, 1, 1)
 n_neighbor_bounds_control(nodetype) =
     error("'n_neighbor_bounds_control' not defined for $nodetype.")
 
+controllablefields(nodetype::Symbol) = controllablefields(Val(nodetype))
```

Comment:
Could also be placed directly underneath the definition of each struct?

Reply:
Agree this isn't so much validation, but I feel like it fits better here than scattered across parameter.jl.